topcoffea/modules/MakeLatexTable.py

```diff
@@ -0,0 +1,103 @@
+# Library of functions for printing a latex table
+# Assumes the dictionary you pass is of this format:
+#   vals_dict = {
+#       key : {
+#           subkey : (val,err)
+#       }
+#   }
+
+# Make the header row
+def format_header(column_lst):
+    s = "\\hline "
+    for i,col in enumerate(column_lst):
```

Comment:
This is probably nitpicky, but it looks like we don't need `enumerate` here, since `i` is never used.

Reply:
it's nitpicky yes, but it's also nice to have a counting variable available to you inside of a `for` loop, but that's probably just a carry over from how I learned C++ (before `auto` and the more fancy iterating constructs became common). However, given the fact that this code is more or less out of the "R&D" stage, if people would prefer to remove it then I see no problem with cutting it.

Reply:
But if you don't use it, then it's a waste to generate it!

Reply:
Ok, thanks, I've removed it (in [this commit](https://github.com/TopEFT/topcoffea/commit/a9ec2233774fe229e1eda961d3a1a5e7f318d8f7)).